tools/dfx-orbit/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 
 [package]
 name = 'dfx-orbit'
-version = '0.4.0'
+version = '0.5.0'
```

Comment:
this should be automatically bumped in the separate release PR

Reply:
I removed the change